engine/src/flutter/lib/ui/window.dart

```diff
@@ -164,10 +164,9 @@ class FlutterView {
   /// See also:
   ///
   ///  * [physicalSize], which returns the current size of the view.
-  // TODO(goderbauer): Wire this up so embedders can configure it. This will
-  //   also require to message the size provided to the render call back to the
-  //   embedder.
-  ViewConstraints get physicalConstraints => ViewConstraints.tight(physicalSize);
+  ViewConstraints get physicalConstraints => _viewConfiguration.viewConstraints.isEmpty
```

Comment:
In what scenarios are the view constraints empty? Can we remove this? :)

Reply:
Looks like tester_main.cc was the correct answer.